packages/airnode-protocol/deployments/references.json

```diff
@@ -14,21 +14,30 @@
     "100": "gnosis",
     "137": "polygon",
     "250": "fantom",
+    "288": "bobaEthereum",
+    "416": "sxNetwork",
     "420": "optimism-testnet",
     "588": "metis-testnet",
     "1088": "metis",
     "1284": "moonbeam",
     "1285": "moonriver",
     "1287": "moonbeam-testnet",
+    "1294": "bobaMoonbeam",
     "2001": "milkomeda",
     "4002": "fantom-testnet",
     "42161": "arbitrum",
+    "42170": "arbitrum-nova",
     "43113": "avalanche-testnet",
     "43114": "avalanche",
+    "43288": "bobaAvalanche",
+    "56288": "bobaBNB",
+    "71402": "nervos",
```

Comment:
?

Reply:
I forgot to update references when I removed `neon-devnet` and rename `nervos`. my bad